Source/WarpX.H

```diff
@@ -156,6 +156,7 @@ public:
     static long nox;
     static long noy;
     static long noz;
+    static bool match_shape_factors;
```

Comment:
```suggestion
```
Let us remove this new parameter and use `l_lower_order_in_v` (renamed as `galerkin_interpolation`)